tools/lxdclient/client.go

```
@@ -123,15 +124,24 @@ func Connect(cfg Config) (*Client, error) {
 		return nil, errors.Trace(err)
 	}
 
-	// If this is the LXD provider on the localhost, let's do an extra check to
-	// make sure the default profile has a correctly configured bridge, and
-	// which one is it.
-	var bridgeName string
-	if remoteID == remoteIDForLocal {
-		bridgeName, err = verifyDefaultProfileBridgeConfig(raw)
+	networkAPISupported := false
+	if cfg.Remote.Protocol != SimplestreamsProtocol {
+		status, err := raw.ServerStatus()
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
+
+		if lxdshared.StringInSlice("network", status.APIExtensions) {
+			networkAPISupported = true
+		}
+	}
+
+	var bridgeName string
+	if remoteID == remoteIDForLocal {
+		// If this is the LXD provider on the localhost, let's do an extra check to
+		// make sure the default profile has a correctly configured bridge, and
+		// which one is it.
+		bridgeName, err = verifyDefaultProfileBridgeConfig(raw, networkAPISupported)
```

We are ignoring the error here, oops